### DISCUSSION TITLE
Copilot multi cursor auto suggestions

### DISCUSSION THREAD

**Discussion by daxaxelrod** [community/community#32619] [Copilot Conversations]
Hi! Love Copilot, and even bought a subscription (which I never do ¯\_(ツ)_/¯). 

When editing multiple lines, copilot recommends a useful autocomplete but when I go to accept it, it doesn't distribute the recommendation across the cursors, it even jumps out of multi-line editing which is a tad annoying. 

Thanks for building a great product, hope to hear from you soon!

**Comment by iamcjwilliams** [comment]
Hello @daxaxelrod Can you post a screenshot of what you're seeing?

**Reply by daxaxelrod** [reply]
Sure! 
<img width="388" alt="Screen Shot 2022-09-12 at 6 35 43 PM" src="https://user-images.githubusercontent.com/10982992/189770316-e837bd6f-f3cb-4cc8-9cb2-65b5f99b4e32.png">
<img width="329" alt="Screen Shot 2022-09-12 at 6 35 50 PM" src="https://user-images.githubusercontent.com/10982992/189770315-14c4c418-0a64-4b76-9823-8397d628c208.png">

**Comment by augustoaraujoo** [comment]
you can't define multiple functions with the same name, are you trying to do that? if it is, it will not be possible

**Reply by daxaxelrod** [reply]
I know that, it was just an example of multi cursor

**Reply by augustoaraujoo** [reply]
ok
